src/test/java/teammates/test/cases/common/FieldValidatorTest.java

```diff
@@ -176,8 +176,7 @@ public void testGetValidityInfoForAllowedName() {
         String nameStartedWithNonAlphaNumChar = "!Amy-Bén s/o O'&|% 2\t\n (~!@#$^*+_={}[]\\:;\"<>?)";
         assertEquals("invalid: typical length with invalid characters", 
                      String.format(INVALID_NAME_ERROR_MESSAGE,
-                                   Sanitizer.sanitizeForHtml(nameStartedWithNonAlphaNumChar),
-                                   typicalFieldName, REASON_START_WITH_NON_ALPHANUMERIC_CHAR, typicalFieldName),
```

Comment:
Reasons no longer needed because the error message will inform the user what is acceptable anyway.

Example of old (I think sort of long-winded error message): `[invalid input] is not acceptable to TEAMMATES as [field name] because it [reason]. The value of [fieldname] should be no longer than [max_length] characters. It should not be empty."`

Since the error message already explain what is acceptable (and there's at most 3 criteria), I think there's no need to specifically tell user what is wrong with the input.